sys/include/net/ng_netconf.h

```diff
@@ -45,7 +45,9 @@ typedef enum {
     NETCONF_OPT_ADDR_LEN,           /**< get the default address length a
                                      *   network device expects as uint16_t in
                                      *   host byte order */
-
+    NETCONF_OPT_SRC_LEN,            /**< get/set the address length to choose
+                                     *   for the network devices source address
+                                     *   as uint16_t in host byte order */
```

Comment:
Sure, but `NETCONF_OPT_ADDR_LEN` is (and all the other primitive type values are) `uint16_t` so I didn't want it to be completely different from the others.